CHANGELOG-developer.next.asciidoc

```diff
@@ -129,6 +129,7 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 - Introduce `libbeat/beat.Beat.OutputConfigReloader` {pull}28048[28048]
 - Update Go version to 1.17.1. {pull}27543[27543]
 - Whitelist `GCP_*` environment variables in dev tools {pull}28364[28364]
+- Add `commandstats` field to Redis module {pull}29662[29662]
```

Comment:
hi @zanyou this is not required here, you can remove, thanks :)

Reply:
I removed it.